packages/server/database/migrations/20221201121536-addHeardSeenRespectedTemplate.ts

```diff
@@ -0,0 +1,113 @@
+import {R} from 'rethinkdb-ts'
+import {PALETTE} from '../../../client/styles/paletteV3'
+
+const createdAt = new Date()
+
+const nameToId = (name: string, type: 'template' | 'prompt') => {
+  const cleanedName = name
+    .replace(/[^0-9a-z-A-Z ]/g, '') // remove emojis and apostrophes
+    .split(' ')
+    .map(
+      (name, idx) =>
+        (idx === 0 ? name.charAt(0).toLowerCase() : name.charAt(0).toUpperCase()) + name.slice(1)
+    )
+    .join('')
+    .trim()
+  return `${cleanedName}${type === 'template' ? 'Template' : 'Prompt'}`
+}
+
+const makeTemplate = (name: string) => ({
+  createdAt,
+  id: nameToId(name, 'template'),
+  isActive: true,
+  name,
+  orgId: 'aGhostOrg',
+  scope: 'PUBLIC',
+  teamId: 'aGhostTeam',
+  type: 'retrospective',
+  updatedAt: createdAt,
+  isStarter: true,
+  isFree: false
+})
+
+type PromptInfo = {
+  question: string
+  description: string
+  groupColor: string
+  templateId: string
+  sortOrder: number
+}
+
+const makePrompt = (promptInfo: PromptInfo) => {
+  const {question, description, groupColor, templateId, sortOrder} = promptInfo
+  return {
+    createdAt,
+    description,
+    groupColor,
+    // FIXME this may create duplicated ids
+    id: nameToId(question, 'prompt'),
```

Comment:
-1 we should not create the ids just based on the question as it can easily create duplicate ids which will overwrite an existing prompt which then is missing (someone even left the FIXME from where you've copied this ;-)
It's not an issue this time, but lets create it from the templateId + question to be sure and create another good example to copy from